clang/lib/CIR/Dialect/IR/CIRDialect.cpp

```diff
@@ -920,6 +921,16 @@ LogicalResult mlir::cir::ComplexImagPtrOp::verify() {
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// LoadOp
+//===----------------------------------------------------------------------===//
+
+void mlir::cir::LoadOp::setAtomic(mlir::cir::MemOrder order) {
+  setMemOrder(order);
+  if (::cir::MissingFeatures::syncScopeID())
```

Comment:
I believe the syncScopeID asserts we have in CIRGen are already enough?

Reply:
I'll add a comment here that the full interface is supposed to be

```
void setAtomic(AtomicOrdering ordering,
                 SyncScope::ID ssid = SyncScope::System);
```